frontend/src/host_orchestrator/orchestrator/createcvdaction.go

```diff
@@ -118,8 +119,13 @@ func (a *CreateCVDAction) launchCVD(op apiv1.Operation) {
 	}
 }
 
+const userArtifactPrefix = "@user_artifact/"
```

Comment:
Let's move this constant to our public API and add the relevant documentation to it. Let's put it closer to the `CreateCVDRequest` struct https://github.com/google/android-cuttlefish/blob/main/frontend/src/liboperator/api/v1/messages.go#L114.

In the context of the public API, I'd recommend constant name: `EnvConfigUserArtifactsVar`, using only the notation literal `@user_artifacts` as the value and leaving the `/` character to the internal replace logic.

Reply:
Moved `userArtifactPrefix` to `messages.go` and renamed it to `EnvConfigUserArtifactsVar`